CHANGELOG.md

```diff
@@ -243,6 +243,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1768**
   - Fix an issue in the stats endpoint in `@cumulus/api` to send back stats for the correct type
 
+- **CUMULUS-1775** 
```

Comment:
This looks like it's under 1.19. Can you please move to the unreleased section?

Reply:
Oops, bad merge presumably.   I'll sort it.

Reply:
Updated: dcfff5b53